tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/TeamMailboxesQuotaExtensionsContract.scala

```diff
@@ -234,7 +234,7 @@ class TeamMailboxesQuotaExtensionsContract {
          |            {
          |                "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
          |                "notFound": [],
-         |                "state": "${INSTANCE.value}",
+         |                "state": "b14eb58e-1975-396c-b362-d6514724ff35",
```

Comment:
Maybe it would be better to just ignore the state value?

Reply:
No need it is fixed value